tests/document-viewer/documentViewer.spec.ts

```diff
@@ -0,0 +1,40 @@
+import {expect, test} from '@playwright/test';
+import * as document_viewer from '../../framework/pages/documentViewer/documentViewer.ts'
+import * as authentication_modal from "../../framework/pages/login.ts";
+import * as header from "../../framework/pages/header.ts"
+import {submitCookieBanner} from "../../framework/pages/cookie-banner.ts";
+import {openLandingPage} from "../../framework/pages/landing.ts";
+
+const documentUrl = process.env.BASE_URL + process.env.BASE_LOCALIZATION + '/document/293381' ;
+const saveAndNavigationDocumentUrl = process.env.BASE_URL + process.env.BASE_LOCALIZATION + '/document/352297';
+
+const userName = process.env.USER_NAME;
+const userPwd = process.env.PASSWORD;
+
+test('View a Book Solution document', async ({page}) => {
+    await openLandingPage(page);
+    await submitCookieBanner(page);
+    await header.clickLoginButton(page, false);
+    await authentication_modal.loginWithEmail(page, userName, userPwd);
+    await document_viewer.openDocument(page, documentUrl);
+    await expect(page).toHaveURL(/\/book-solution/);
+});
+
+test('Cannot use page navigation on a 1 page document', async ({page}) => {
+    await openLandingPage(page);
+    await submitCookieBanner(page);
+    await header.clickLoginButton(page, false);
+    await authentication_modal.loginWithEmail(page, userName, userPwd);
+    await document_viewer.openDocument(page, saveAndNavigationDocumentUrl);
+    await expect(document_viewer.checkPreviousButtonState(page)).toBeFalsy();
+    await expect(document_viewer.checkNextButtonState(page)).toBeFalsy();
```

Comment:
```suggestion
    await expect(await document_viewer.checkPreviousButtonState(page)).toBeFalsy();
    await expect(await document_viewer.checkNextButtonState(page)).toBeFalsy();
```
otherwise `document_viewer.checkNextButtonState(page)` returns an empty object and test fails on those checks